docs/sphinx/source/strictdoc.rst

```diff
@@ -519,6 +519,42 @@ Special feature of ``[COMPOSITE_REQUIREMENT]``: like ``[SECTION]`` element, the
 often, a more basic combination of nested ``[SECTION]`` and ``[REQUIREMENT]``
 elements should do the job.
 
+Include Files
+~~~~~~~~~~~~~
+
+Strictdoc ``.sdoc`` files can be built-up from including other fragment documents.
+
+The ``[INCLUDE_SECTION_FROM_FILE]`` element can be used anywhere a ``[SECTION]`` can
+be used and will evaluate to a Section containing everything in the file referenced
+by its ``FILE:`` property. The files included must start with a ``[FRAGMENT]`` directive
+but are otherwise identical to ``*.sdoc`` files. They can have any filename except a
+``.sdoc`` extension.
+
+Here is an example pair of files identical to the section example above. First the
+``.sdoc`` file has a ``[INCLUDE_SECTION_FROM_FILE]`` that references the latter file.
+
+.. code-block:: text
+
+    [DOCUMENT]
+    TITLE: StrictDoc
+
+    [INCLUDE_SECTION_FROM_FILE]
+    FILE: include.ssec
+
+Then the referenced file, ``include.ssec``:
+
+.. code-block:: text
+
+    [FRAGMENT]
```

Comment:
In the https://github.com/strictdoc-project/strictdoc/pull/632, I have tried to work around the `Dual-purposing [SECTION] as the include target and a section-as-usual` problem that you mentioned by making it explicit what's a fragment (`FRAGMENT`) and its contents (the usual `SECTIONS`).

So the example below should have:

```
[FRAGMENT]

[SECTION]
TITLE: Free text

[/SECTION]
```